errors/IncorrectAnonymousArgument.md

```diff
@@ -5,31 +5,54 @@
 ```purescript
 module Example where
 
-add = (_ + _)
```

Comment:
This `add` example is problematic because:
- Not ideal to overwrite Prelude functions, while also using the corresponding infix operator.
- Even with a different name, it's trivially replaced with `add` or `(+)`.
- Unclear whether the intention in the broken case is to reuse the same anonymous arg, or consider these separate args. And if separate args, is the expectation that they are populated in order?